packages/server-core/src/projects/project/project-helper.ts

```diff
@@ -1517,6 +1518,7 @@ export const updateProject = async (
     )
   }
   // run project install script
+  await execPromise(`npm install`, { cwd: appRootPath.path })
```

Comment:
i wonder if instead of this, we should only do this after (or as part of) a successful build, since if the project hook relies on services it specifies, it would throw errors (that could result in worse problems)

Reply:
I may have slightly misidentified the problem in the description. The problem we're encountering is that projectEventHooks in the MT project imports a file that imports some AWS SDKs that are only in the MT project, not in the core engine. onInstall doesn't use them, nor is it calling any of the services that are part of the project, but the import itself fails because of those downstream missing dependencies. I didn't see that problem when I ran this branch on QAT.

Reply:
Yeah okay, I am identifying a whole class of issues we may run into in the future. This solution is a good stopgap for now.